python/pyspark/mllib/feature.py

```diff
@@ -164,6 +163,13 @@ class StandardScaler(object):
     variance using column summary statistics on the samples in the
     training set.
 
+    :param withMean: False by default. Centers the data with mean
+                     before scaling. It will build a dense output, so this
+                     does not work on sparse input and will raise an
+                     exception.
+    :param withStd:  True by default. Scales the data to unit
```

Comment:
We don't do vertical alignment in Spark code. So we should remove one space after `withStd:`.